app/controllers/api/v1/organizations_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Api::V1::OrganizationsController < ApplicationController
+
+  def create
+    @organization = Organization.create(organization_params)
+    render json: @organization
+  end
+
+  private
+    def organization_params
+      params.require(:organization).permit(:id, :image, :name, :address, :phone, :email, :welcomeText, :aboutUsText)
+    end
```

Comment:
think CRUD don't go yet

Reply:
Yes. I didn't do the entire CRUD, I just add this part because I was following the instructions of how to add an image with active storage

Reply:
I agree with @DevChristians, de controllers must be empty at this phase of the project.

Reply:
Ok! I'll change it